backend/app/models/model.py

```diff
@@ -0,0 +1,65 @@
+import os
+import re
+from typing import List, Dict, Union
+from dataclasses import dataclass, asdict
+import nltk
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+
+
+@dataclass
+class CleanedData:
```

Comment:
Where would this class be called? I understand ResumeDataCleaner, but what about this?